src/TransactionBuilder.cs

```diff
@@ -654,7 +654,8 @@ private LineItemModel GetMostRecentLine(string memberName = "")
         /// <returns></returns>
         public async Task<TransactionModel> CreateAsync()
         {
-            return await _client.CreateTransactionAsync(null, _model);
+            return await _client.CreateTransactionAsync(null, _model)
+                .ConfigureAwait(false);
```

Comment:
Hmm, I notice our standard pattern should really have ConfigureAwait(false) on every Async line.  I'll get that incorporated right away.